carsus/base.py

```diff
@@ -30,7 +30,7 @@ def init_db(fname=None, **kwargs):
     """
     print "Initializing the database"
 
-    url = "sqlite:////" + fname if fname is not None else "sqlite://"
+    url = "sqlite:///" + fname if fname is not None else "sqlite://"
```

Comment:
my suggestions is to get rid of this and just have db url. Then people can do that themselves (including different databases).